demo/wdio.conf.js

```diff
@@ -22,13 +12,14 @@ const config = {
   // Custom settings
   // ===============
   logLevel: 'info', // trace | debug | info | warn | error | silent
-  outputDir: './_results_',
+  outputDir: path.join(process.cwd(), `./_results_`),
```

Comment:
Let's better use `__dirname` instead of `process.cwd()`

Reply:
We can not use __dirname in ESM modules, sry :)

Reply:
you can: https://blog.logrocket.com/alternatives-dirname-node-js-es-modules/

Reply:
This is an example. Humans are lazy (and I think it is okay). Highly likely this example config will be copy-pasted "as is". 
In my practice devs usually place configs to wdio/jest/webpack to some separate folder, containing different config options depending on running environment. F.e. (and I did the same) - disable firefox for local development, but enable it in CI / Jenkins. In that case outputDir will point to the dir, where configs are stored. 
`process.cwd()` is actually exactly the same as relative path `./_results_`; but using `__dirname` here will change the behavior, and may lead to unexpected (unexpected by user) results.  
If you insist - I can change it to url parsing from `import.meta.url`, but personally I don't see the point.

Reply:
Using `__dirname` makes the behavior consistent.

> process.cwd() is actually exactly the same as relative path ./_results_;

This should be changed too then, if you provide a path within a config file it always will be relative to the config file and not relative to your CWD. I've made this mistake with previous WebdriverIO versions and got around to fix it in v8.

I see your point however I would prefer consistency and using `__dirname` gives us that. People can still change it to `process.cwd()` if they know what they do.